docs/modules/ROOT/pages/getting-started/relationships-and-advanced-filtering.adoc

```diff
@@ -1,41 +1,46 @@
-= Relationships and complex filtering
+= Relationships and advanced filtering
 
-This tutorial shows how to add relationships and more complex filters, and alias the results of the projection of the Cypher query from the xref:getting-started/filters-and-projections.adoc[previous tutorial].
+This tutorial shows how to add relationships and more advanced filters, as well as alias the results of the projection.
+It uses the Cypher query example from the xref:getting-started/filters-and-projections.adoc[Filters and projections] tutorial and the link:https://neo4j.com/docs/getting-started/appendix/example-data/#built-in-examples[Movies Dataset].
```

Comment:
It is a bit confusing the part `It uses the Cypher query example from...`

This is more of building up from that example

Reply:
But building up from the example isn't using it, in a way? What if it says 

```suggestion
It extends the Cypher query example from the xref:getting-started/filters-and-projections.adoc[Filters and projections] tutorial and the link:https://neo4j.com/docs/getting-started/appendix/example-data/#built-in-examples[Movies Dataset].
```